### PR TITLE
 Enhanced 'build_apk.py' logic & 'README.md'

### PR DESCRIPTION
**Pull request by fangxingSR**
Enhanced 'build_apk.py' logic & 'README.md'.

    There was an issue in 'build_apk.py': when only 'qnnsdk' and 'model_path' were provided as inputs, it incorrectly invoked the 'printmenu()' function to display a model selection menu, despite the model always being renamed to 'classification.tflite'. To address this, I adjusted the logic so that the model selection menu is only printed when neither 'model_name' nor 'model_path' is specified by the user. Specifically, I changed line 45 of 'build_apk.py' to read:
    ```
    if not args.model_name and not args.model_path:
    ```
    ﻿
    Additionally, I made several descriptive improvements to the 'README.md' file. Recognizing that newcomers primarily aim to compile and install the ImageClassification application, I included more concise and efficient instructions tailored to their needs, aiming to minimize their trial-and-error process and facilitate a smoother onboarding experience."

### CODE REVIEW

**Review by kory — APPROVED** [review]

**Comment by kory** [comment]
Thank you!

File: apps/android/ImageClassification/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 1. Java, android-sdk and sdkmanager is already set at user's end
 2. User should have Linux QNN SDK in local machine.
+3. If models need to be downloaded during the execution of build_apk.py, qai-hub must be installed first according to the[ README.md in ai-hub-models](https://github.com/quic/ai-hub-models/blob/main/README.md).
```
Comment (kory) [inline apps/android/ImageClassification/README.md:5]: Can you replace `qui-hub` with `AI Hub Models`

Comment (kory) [inline apps/android/ImageClassification/README.md:5]: Could you also move the space before `README.md` outside of the link text. Thanks!

Comment (fangxingSR) [reply apps/android/ImageClassification/README.md]: Thank you for your correction. The document has been revised

Comment (fangxingSR) [reply apps/android/ImageClassification/README.md]: This was my negligence, thank you for pointing out the mistake.

**Comment by kory** [comment]
Thanks! Your contribution is included in release 0.6.0, which is being pushed out today.

**Pull request merged**